### PR TITLE
feat: make client features endpoint memoizable

### PR DESCRIPTION
**Pull request by chriswk**

### CODE REVIEW

File: src/lib/routes/client-api/feature.js [hunk 1]
```diff
@@ -16,20 +18,45 @@ const FEATURE_COLUMNS_CLIENT = [
...
 class FeatureController extends Controller {
-    constructor({ featureToggleService }, getLogger) {
+    constructor({ featureToggleService }, { getLogger, experimental }) {
         super();
         this.toggleService = featureToggleService;
         this.logger = getLogger('client-api/feature.js');
         this.get('/', this.getAll);
         this.get('/:featureName', this.getFeatureToggle);
+        if (experimental) {
+            if (experimental.clientFeatureMemoize) {
```
Comment (ivarconr) [inline src/lib/routes/client-api/feature.js:28]: one level? 

```
if (experimental && experimental.clientFeatureMemoize) {
...
```

File: src/lib/routes/client-api/feature.test.js [hunk 2]
```diff
@@ -40,6 +42,52 @@ test('should get empty getFeatures via client', t => {
         });
 });
 
+test('if caching is enabled should memoize', t => {
```
Comment (ivarconr) [inline src/lib/routes/client-api/feature.test.js:45]: :+1:

**Review by ivarconr — APPROVED** [review]

File: src/lib/routes/client-api/feature.js (hunk 1 above)
Comment (chriswk) [reply src/lib/routes/client-api/feature.js:28]: Agreed

Comment (chriswk) [reply src/lib/routes/client-api/feature.js]: And fixed :)

**Pull request merged**